docker/Dockerfile.torizon-demos

```diff
@@ -37,7 +37,7 @@ ENV PKG_CONFIG_ALLOW_CROSS=1
 COPY . /slint
 WORKDIR /slint
 RUN mkdir demos \
-    && for demo in printerdemo slide_puzzle gallery opengl_underlay carousel todo energy-monitor; do \
+    && for demo in printerdemo-bin slide_puzzle gallery opengl_underlay carousel todo-bin energy-monitor; do \
     cargo build --release --target $RUST_TOOLCHAIN_ARCH --features slint/renderer-skia,slint/backend-linuxkms-noseat -p $demo || exit 1; \
```

Comment:
This uses -p but does it need to be --bin after the rename?

Reply:
Yes, but --bin only works for item in the default-member in the workspace. Which many demos are not part of.

So the logic here needs to be made more complicated since the argument to -p is not going to be the binary name